cli/src/commands/node/mod.rs

```diff
@@ -158,9 +166,17 @@ impl Node {
             )
         });
 
+        let config_file = self.config.map(File::open).transpose()?;
```

Comment:
If incorrect file will be specified, something like `[ERROR] No such file or directory (os error 2)`, with no clue which file is not found. I would use `anyhow::Context` here, but for some reason we're using boxed Error in `cli`, so probably an `error` log entry is worth adding here.

Reply:
Does this look better:
```
2024-05-09T09:19:20.509149Z ERROR openmina::commands::node: time: "1715246360507319345", kind: "ConfigFileError", summary: "failed to open config file", error: "Os { code: 2, kind: NotFound, message: \"No such file or directory\" }"
[ERROR] No such file or directory (os error 2)
```
?